content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -116,6 +146,7 @@ The CSV name currently installed must be in the `skips` field of a CSV in the ne
 
 The version currently installed must be in the `olm.skipRange` field of a CSV in the new channel.
 
+
```

Comment:
Minor nit as I doubt this would show up in the preview, but this looks like an unneeded newline that markdown linters will complain about locally.

Reply:
Yes, removed that new line.